includes/class-wau-admin-settings.php

```diff
@@ -113,6 +121,32 @@ public function wau_settings_enable_renderer() {
 			<?php
 		}
 
+		/**
+		 * Display HTML do not send link in email
+		 */
+
+		public function wau_settings_enable_renderer_2() {

```

Comment:
Please use logical name instead of `wau_settings_enable_renderer_2`

Use a name which corresponds to the function working.